server/src/main/java/org/elasticsearch/index/shard/IndexShard.java

```diff
@@ -430,6 +436,10 @@ public BulkOperationListener getBulkOperationListener() {
         return this.bulkOperationListener;
     }
 
+    public Releasable startTrackingBulkOperationTime() {
```

Comment:
Can we add verification to integration tests that these are always released, i.e., that for every `IndexShard`, the set of of operations end up empty eventually?